rasa_core/policies/keras_policy.py

```diff
@@ -217,6 +233,7 @@ def persist(self, path: Text) -> None:
             meta = {"model": "keras_model.h5",
                     "epochs": self.current_epoch}
 
+            meta.update(self._tf_config)
```

Comment:
as far as I understand `self._tf_config` is `tf.ConfigProto` and not a `dict`?

Reply:
about this problem I will extract in `policy`.

Reply:
I persist `tf.ConfigProto` with pickle in `keras_policy`.